src/main/java/com/faforever/client/player/PlayerInfoWindowController.java

```diff
@@ -401,7 +425,9 @@ public void onRatingTypeChange() {
     if (ratingTypeComboBox.getValue() != null) {
       ratingHistoryChart.setVisible(false);
       loadingHistoryPane.setVisible(true);
-      loadStatistics(ratingTypeComboBox.getValue()).thenRun(() -> JavaFxUtil.runLater(this::plotPlayerRatingGraph));
+      if (player != null) {
+        loadStatistics(ratingTypeComboBox.getValue()).thenRun(() -> JavaFxUtil.runLater(this::plotPlayerRatingGraph));
+      }
```

Comment:
I can't explain it, but when I leave this check out I regularly get NPEs there when opening the info window. It's very weird because the player should get set in line 210.

Reply:
Ah it is probably because the rating type gets set before the player is populated in the initialization